src/Riok.Mapperly.Abstractions/MapperAttribute.cs

```diff
@@ -99,4 +99,11 @@ public class MapperAttribute : Attribute
     /// Determines the access level of members that Mapperly will map.
     /// </summary>
     public MemberVisibility IncludedMembers { get; set; } = MemberVisibility.AllAccessible;
+
+    /// <summary>
+    /// Controls the priority of constructors used in mapping.
+    ///  When enabled, a parameterless constructor is prioritized over constructors with parameters.
+    ///  When disabled, accessible constructors are ordered in descending order by their parameter count.
```

Comment:
Nit: I'd use `<c>true</c>` and `<c>false</c>` instead of "enabled"/"disabled" as it directly links to the value.
Is there a reason for the space at the beginning of the line?
The same applies to the comment on `MapperConfiguration`.

Reply:
done